src/components/dialogs/parameters/sensi/sensitivity-table.tsx

```diff
@@ -68,45 +64,36 @@ const SensitivityTable: FunctionComponent<SensitivityTableProps> = ({
                 border: 'solid 0px rgba(0,0,0,0.1)',
             }}
         >
-            <Table stickyHeader size="small">
+            <Table stickyHeader size="small" sx={{ tableLayout: 'fixed' }}>
                 <TableHead>
-                    <TableRow>
-                        {columnsDefinition.map((column: any) => (
-                            <TableCell>
-                                <Box
-                                    sx={{
-                                        backgroundColor: column.color,
-                                    }}
-                                >
-                                    <FormattedMessage id={column.label} />
-                                </Box>
-                            </TableCell>
-                        ))}
-                        <TableCell>
-                            <Tooltip
-                                title={intl.formatMessage({
-                                    id: 'AddRows',
-                                })}
+                    {columnsDefinition.map((column: any) => (
+                        <TableCell
+                            key={column.dataKey}
+                            sx={{ width: column.width, textAlign: 'center' }}
+                        >
+                            <Box
+                                sx={{
+                                    justifyContent: 'space-between',
+                                    alignItems: 'center',
+                                    fontWeight: 'normal',
+                                    margin: 1,
```

Comment:
- font-weight: design will not be the same as other components using Table Cell, I think  that you will not need to modify this.
- margin: it is not necessary

anyway, I don't think that this box is necessary, even if it is necessary, applying "justifyContent" and "alignItem" outside a flex context (display: flex) will not have an effect on style.

Reply:
done